server/platform/services/telemetry/telemetry.go

```diff
@@ -93,6 +93,17 @@ const (
 	TrackPlugins  = "plugins"
 )
 
+type TrackSKU string
+
+const (
+	TrackProfessionalSKU TrackSKU = "professional"
+	TrackEnterpriseSKU   TrackSKU = "enterprise"
+)
+
+type TrackFeature string
+
+const TrackGuestFeature TrackFeature = "guest"
```

Comment:
Minor, perhaps could be renamed to `guest_accounts`.

Reply:
renamed, thanks!